core/contracts/financial-templates/implementation/Liquidatable.sol

```diff
@@ -422,6 +369,39 @@ contract Liquidatable is PricelessPositionManager {
         return _getCollateral(liquidations[sponsor][uuid].rawLockedCollateral);
     }
 
+    /**
+     * @notice This settles a liquidation if it is in the PendingDispute state. If not, it will immediately return.
+     * If the liquidation is in the PendingDispute state, but a price is not available, this will revert.
+     * @param id to uniquely identify the dispute to settle
+     * @param sponsor the address of the sponsor who's dispute is being settled
+     */
+    function _settle(uint id, address sponsor) internal {
+        LiquidationData storage liquidation = _getLiquidationData(sponsor, id);
+
+        // If this dispute cannot be settled, just return.
+        if (liquidation.state != Status.PendingDispute) {
+            return;
```

Comment:
I don't think you need this check anymore, in fact you might not even need `PendingDispute` in the State enum anymore